### DISCUSSION TITLE
Where to find gem API documentation for Pro features?

### DISCUSSION THREAD

**Discussion by marten** [rmosolgo/graphql-ruby#4264] [Q&A]
On https://graphql.pro/ there is a link that claims "GraphQL::Pro documentation can be found with the [GraphQL Ruby gem documentation](https://graphql-ruby.org/)." but it just links to the homepage. In the documentation under https://graphql-ruby.org/api-doc/2.0.15/ I don't seem to see any (e.g.) Pundit-related things.

**Comment by rmosolgo** [comment]
Hi! Yes, the usage documentation for GraphQL-Pro is on the graphql-ruby website. The API docs aren't published on line (that would publish the source, too), but you can generate it locally with: 

```
# get the installed location of graphql-pro 
bundle show graphql-pro 
# navigate to that directory
cd  .... 
# start up YARD 
yard server 
```

Then, you can open your browser to view the locally-generated docs: http://0.0.0.0:8808/docs/GraphQL/Pro

Let me know how that works for you! Since those docs aren't actually published, I haven't polished them up too much; let me know if there's something in particular you're looking for and I'd be happy to help.
